src/components/Table/Table.tsx

```diff
@@ -13,9 +13,8 @@ const Container = styled.table`
   background-color: ${({ theme }) => theme.palette.background.default};
 `;
 
-export type Props = {
-  children: React.ReactNode;
-};
+export type Props = React.ComponentPropsWithRef<"table"> &
+  Partial<{ verticalSpacing: VerticalSpacing }>;
```

Comment:
```suggestion
  { verticalSpacing?: VerticalSpacting };
```

他のコンポーネントと合わせるとこの書き方が良さそうですね！